src/atc_api/main.go

```diff
@@ -32,9 +33,13 @@ func main() {
 		api.Log.Println("Connected to atcd socket on", args.ThriftAddr)
 	}
 
+	if args.IPv4 == "" || args.IPv6 == "" {
+		api.Log.Fatalln("You must provide both -4 and -6 arguments to run the API.")
```

Comment:
We want them to provide one or more of `IPv4` or `IPv6`. They can provide both, they can provide either. If they don't provide either, we should bail.